src/patternfly/components/Select/deprecated/Select.css

```
@@ -0,0 +1,55 @@
+#ws-html-deprecated-c-select-single-with-top-expanded,
+#ws-html-deprecated-c-select-single-expanded,
+#ws-html-deprecated-c-select-single-expanded-and-selected,
+#ws-html-deprecated-c-select-single-with-typeahead-expanded-and-selected,
+#ws-html-deprecated-c-select-plain-toggle-expanded,
+#ws-html-deprecated-c-select-view-more-menu-item,
+#ws-html-deprecated-c-select-loading-menu-item,
+#ws-html-deprecated-c-select-placeholder-expanded,
+#ws-html-deprecated-c-select-placeholder-item-disabled,
+#ws-html-deprecated-c-select-placeholder-item-enabled {
```

@ashley-o0o noticed an example is missing a height today - https://patternfly-pr-5667.surge.sh/components/menus/select#multi-with-typeahead-invalid

A wee bit of scope creep, but could we sneak that fix in here? The ID for that example is `ws-html-deprecated-c-select-multi-with-typeahead-invalid`

```suggestion
#ws-html-deprecated-c-select-placeholder-item-enabled,
#ws-html-deprecated-c-select-multi-with-typeahead-invalid {
```

---

sure thing